styles.css

```diff
@@ -74,15 +77,64 @@ a:active {
 
 .navbar__container {
   width: 100%;
-  margin: 20px;
-  position: fixed;
 }
 
 .navbar__hamburger {
   width: 25px;
+  margin: 20px 0 10px 20px;
+  position: fixed;
+  z-index: 9;
+}
+
+.navbar__hamburger.active {
+  display: none;
+}
+
+.navbar__links {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
```

Comment:
- [ ] The nav link menu should have the similar spacing as the original design. 
![image](https://user-images.githubusercontent.com/71874425/163296135-5b93c59a-b169-4d61-99ce-847afa9c3d58.png)
- Please, Kindly fix it.